contrib/babelfishpg_tds/src/backend/tds/tdstypeio.c

```diff
@@ -315,10 +329,20 @@ TdsAnyToServerEncodingConversion(Oid oid, pg_enc enc, char *str, int len)
 	Oid 		typioparam;
 	char 		*pstring;
 	Datum 		pval;
+	int			actualLen;
 
 	getTypeInputInfo(oid, &typinput, &typioparam);
```

Comment:
We can call this also only once and store typinput and typioparam for the associated string datatype rather than doing lookup again and again.

Reply:
Removed the redundant call to getTypeInputInfo function